core/trino-main/src/main/java/io/trino/operator/aggregation/listagg/ListaggAggregationState.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.operator.aggregation.listagg;
+
+import io.airlift.slice.Slice;
+import io.trino.spi.block.Block;
+import io.trino.spi.function.AccumulatorState;
+
+public interface ListaggAggregationState
+        extends AccumulatorState
+{
+    void setSeparator(Slice separator);
+
+    Slice getSeparator();
+
+    void setOverflowTruncationFiller(Slice overflowTruncationFiller);
```

Comment:
This is a mouthful. I'd simplify to just `filler`

Reply:
Changed to `overflowFiller`.